docs/reference/indices/split-index.asciidoc

```diff
@@ -196,9 +215,36 @@ split process begins. When the split operation completes, the shard will
 become `active`. At that  point, Elasticsearch will try to allocate any
 replicas and may decide to relocate the primary shard to another node.
 
-[float]
-==== Wait For Active Shards
+
+[[split-wait-active-shards]]
+==== Wait For active shards
```

Comment:
Lowercase for?

Reply:
Good catch. Fixed with 6f981b96518da9cef9c3df87e5a546303f42b0f6.